R/MsBackendHdf5Peaks.R

```diff
@@ -172,30 +190,16 @@ setReplaceMethod("mz", "MsBackendHdf5Peaks", function(object, value) {
              "(i.e. peaksCount(object))")
     pks <- mapply(cbind, mz=value, intensity=ints,
                   SIMPLIFY = FALSE, USE.NAMES = FALSE)
-    peaks(object) <- pks
-    object
+    replaceList(object, pks)
 })
 
 #' @rdname hidden_aliases
-setMethod("peaks", "MsBackendHdf5Peaks", function(object) {
-    if (!length(object))
-        return(list())
-    fls <- unique(object@spectraData$dataStorage)
-    if (length(fls) > 1) {
-        f <- factor(dataStorage(object), levels = fls)
-        unsplit(bpmapply(
-            FUN = .h5_read_peaks,
-            fls,
-            split(scanIndex(object), f),
-            object@modCount,
-            SIMPLIFY = FALSE, USE.NAMES = FALSE, BPPARAM = bpparam()),
-            f)
-    } else
-        .h5_read_peaks(fls, scanIndex(object), object@modCount)
+setMethod("peaksCount", "MsBackendHdf5Peaks", function(object) {
```

Comment:
Should this be `lengths` or so? Instead of **peaks**Count?

Reply:
Sure! Yes! We said that. I will remove the `peaksCount` method.

Reply:
Hm, only thing is that there's no generic `lengths`.